packages/forms/stories/combobox.stories.tsx

```diff
@@ -184,13 +184,13 @@ export const CustomFilterHandler = () => {
 };
 
 export const SingleChoiceWithForm = () => {
-  const { register } = useForm<{ fruit: string }>();
+  const { register, getValues } = useForm<{ fruit: string }>({ defaultValues: { fruit: 'lime' } });
 
   return (
     <div className="h-[40rem]">
       <FormControl>
         <FormLabel>Favoritfrukt</FormLabel>
-        <Combobox placeholder="Välj en frukt" {...register('fruit')}>
+        <Combobox placeholder="Välj en frukt" {...register('fruit')} defaultValue={getValues('fruit')}>
```

Comment:
thinking of using formState.defaultValues.fruit here instead, as getValues will trigger a change for the defaultValue-prop everytime onChange is run